browserslist

```diff
@@ -0,0 +1,6 @@
+last 2 Chrome versions
+last 2 Firefox versions
+last 2 Safari versions
+last 2 Edge versions
+IE 11
```

Comment:
Let's remove IE 11 here, because I don't want to commit to formally supporting it with the exported files. We've already been using things like `Array.includes` which I think are unsupported?

Reply:
That's where this gets tricky. Currently it is possible to use Slate in IE11 by including `babel-polyfill` to account for unsupported primitives -- any ES6 syntax is fully converted to ES5. And also, Slate seems to generally work in IE11 (and there's a small handful of open issues which might get worked at some point to fix the bugs that do exist). If we remove IE 11 here then Slate will almost certainly break completely in IE since we wouldn't be transpiling ES6 syntax that IE doesn't support. So I strongly feel that we should keep it in to allow continued development there.

Perhaps it would be helpful to add a comment to the `browserslist` to indicate that this is not a formal list of browsers we officially support but rather a broader set of browsers that we're willing to transpile ES6 features for? (Kind of verbose but hopefully you get what I mean.)